.bin/esy

```diff
@@ -26,20 +28,28 @@ SCRIPTDIR="$( cd -P "$( dirname "$SOURCE" )" && pwd )"
 # Capturing stderr is very important to prevent nodejs from setting
 # stderr to nonblocking mode
 
+EJECT_PATH="$PWD/node_modules/.cache/esy"
+
+esy-eject () {
+  mkdir -p `dirname $EJECT_PATH`
+  rm -rf $EJECT_PATH
```

Comment:
We eject on every invokation now but really could just check `node_modules` and `package.json` mtimes.